libs/ngxtension/inject-document-visibility/src/inject-document-visibility.ts

```diff
@@ -51,10 +51,10 @@ export function injectDocumentVisibility(
 
 		const docVisible$: Observable<DocumentVisibilityState> = fromEvent(
 			doc,
-			'visibilitychange',
+			'visibilitychange'
 		).pipe(
 			startWith(doc.visibilityState),
-			map(() => doc.visibilityState),
+			map(() => doc.visibilityState)
```

Comment:
Hi @fiorelozere - I see minor Prettier issues here. Could you please run `nx format:write` locally (with the latest node modules)? This might be an existing miss prior to your changes.